File: Two Sum.c
C Language :

int *sum(int* nums,int target,int numsSize,int* returnSize)
{
    int* arr=malloc(sizeof(int*));
    *returnSize=2;
    for(int i=0;i<numsSize-1;i++)
    {
      for(int j=i+1;j<numsSize;j++)
      {
       if(nums[i]+nums[j]==target)
       {
         arr[0]=i;
         arr[1]=j;
       }
      }
    }
   return arr;
}

File: Two Sum.java
Java Language :

class Solution {
    public int[] twoSum(int[] nums, int target) {
       HashMap<Integer,Integer> hm = new HashMap<>();
       for(int i=0;i<nums.length;i++){
        int num1=target-nums[i];
        if(hm.containsKey(num1)){
            return new int[]{hm.get(num1),i};
        }
        hm.put(nums[i],i);
       }
       return new int[]{};
    }
}
